react/.eslintrc

```diff
@@ -5,15 +5,29 @@
         "plugin:import/errors",
         "plugin:import/warnings",
         "plugin:jsx-a11y/recommended",
-        "plugin:import/typescript"
+        "plugin:import/typescript",
+        "prettier"
     ],
     "plugins": ["jsx-a11y"],
+    "settings": {
+        "import/resolver": {
+            "node": {
+                "extensions": [".ts", ".tsx"],
+                "moduleDirectory": ["src", "node_modules"]
+            }
+        }
+    },
     "rules": {
-        "max-len": ["warn", { "ignoreComments": true }],
+        "max-len": ["warn", { "ignoreComments": true, "code": 200, "tabWidth": 4 }],
```

Comment:
Hej @vahidbazzaz , can you explain why you made the adjustments to the rules? Did it turn out it does not match our requirements?

Reply:
@pylipp Thank you for reviewing the PR. To make it more readable, I think we should set `code` length to `100` and reduce the `tabWidth` to 2. What do you think? I will update the documents if you agree with this change

Reply:
I won't work in the FE codebase much, so I don't have that much of an opinion. Makes sense what you're saying though, go ahead!